### DISCUSSION TITLE
trigger handle_on when an arg is False

### DISCUSSION THREAD

**Discussion by Mathanraj-Sharma** [h2oai/wave#752] [Q&A] [answered]
Hi, I want to trigger the show_cards() handler when the show_cards arg is False. How can I do this?

```
@on()
async def show_cards(q: Q):
    await toggle_to_card_view(q)
```

**Comment by lo5** [comment]
@Mathanraj-Sharma - which component is named `show_cards` in your app? Is it a checkbox?

**Comment by vopani** [comment, marked as answer]
@Mathanraj-Sharma This is now possible using `predicate` in `@on` like shown here: https://github.com/h2oai/wave/issues/787
